### PR TITLE
added noise to dataman.TRCFile class

### PR DESCRIPTION
**Pull request by jimmyDunne**

### CODE REVIEW

File: perimysium/dataman.py [hunk 1]
```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
 
         f.close()
 
+    def addnoise(self, noiseWidth):
```
Comment (chrisdembia) [inline perimysium/dataman.py]: Can you rename this as `add_noise`? See the python style guidelines: https://www.python.org/dev/peps/pep-0008/#function-names

Comment (chrisdembia) [inline perimysium/dataman.py]: Oh and I don't think you want to return `self`, do you? You can just continue using the object.

Comment (chrisdembia) [inline perimysium/dataman.py]: Also `noiseWidth` -> `noise_width`.

Comment (jimmyDunne) [inline perimysium/dataman.py]: I initially thought that a user may not want to overwrite the original object, rather clone and do the operations.

File: perimysium/dataman.py [hunk 2]
```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
...
+        for imarker in range(self.num_markers):
+
+            # create the noise object
+            noise = np.random.normal(0, noiseWidth, self.num_frames)
+            # add noise the marker existing marker data.
+            self.data[self.marker_names[imarker] + '_tx'] += noise
```
Comment (tkuchida) [inline perimysium/dataman.py]: Perhaps the noise added to each coordinate should be different? Also, "the marker" should be "to the" in the comment. :horse:

Comment (chrisdembia) [inline perimysium/dataman.py]: oh hey Tom! :frog:

File: perimysium/dataman.py [hunk 3]
```diff
@@ -409,6 +409,21 @@ def write(self, fpath):
...
+    def add_noise(self, noise_width):
+        """ add random noise to each component of the marker trajectory
+            The noise mean will be zero, with the noise_width being the
+            standard deviation.
+
+            noise_width : int
+        """
+        for imarker in range(self.num_markers):
+            components = ['_tx', '_ty', '_tz']
+            for iComponent in range(0, 3):
```
Comment (chrisdembia) [inline perimysium/dataman.py]: fyi you can omit that first 0; `range(3)` would do the same thing.

Comment (chrisdembia) [inline perimysium/dataman.py]: no need to make this change though.

**Comment by chrisdembia** [comment]
Let me know if this is ready to be merged.

**Comment by jimmyDunne** [comment]
thanks for the feedback, I'm still learning so I do appreciate feedback on good python coding structures. If there are better ways for me to do things, please let me know

**Pull request merged**